src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -341,6 +341,13 @@ namespace service_nodes
                 }
                 else {
                   int64_t credit = calculate_decommission_credit(info, latest_height);
+                   if (m_core.get_nettype() == cryptonote::MAINNET && credit < 0)
+                   {
+                     static uint64_t hf13_height = m_core.get_earliest_ideal_height_for_version(cryptonote::network_version_13_enforce_checkpoints);
```

Comment:
It would be nice to make this generic to always apply to the highest known fork so that we don't have to update it for HF14, etc.

Reply:
Good point, updated.